easybuild/easyconfigs/f/flex/flex-2.5.35-goolf-1.4.10.eb

```diff
@@ -8,7 +8,11 @@ sometimes called a tokenizer, is a program which recognizes lexical patterns in
 toolchain = {'name': 'goolf', 'version': '1.4.10'}
 toolchainopts = {'pic': True}
 
-sources = [SOURCELOWER_TAR_GZ]
-source_urls = ['http://prdownloads.sourceforge.net/%(namelower)s']
+sources = ['flex-2-5-35.tar.gz']
+source_urls = ['https://github.com/westes/flex/archive/']
+
+builddependencies = [('Autotools', '20150215')]
```

Comment:
@migueldiascosta I'd stick with using the `dummy` toolchain for this dep, to avoid needing to add more easyconfigs to resolve these deps

```python
builddependencies = [('Autotools', '20150215', '', True)]
```